config/manifests/bases/sandboxed-containers-operator.clusterserviceversion.yaml

```diff
@@ -13,6 +13,13 @@ metadata:
         }
       ]
     capabilities: Seamless Upgrades
+    features.operators.openshift.io/disconnected: "true"
+    features.operators.openshift.io/fips-compliant: "false"
```

Comment:
nit: this should be true given
`operators.openshift.io/infrastructure-features: '["disconnected", "fips"]'`

Reply:
ProdSec invalidated our FIPS compliance by invalidating anything based On UBI builds.  We must build on rhel-els base images for FIPS compliance now.  I wasn't sure what to do here.  This pr is about resolving the CVP issues and it doesn't care what value is set, just that it is set.  I think it would be clearer if we create additional stories to resolve the FIPS compliance issue.